cachito/workers/config.py

```diff
@@ -140,6 +185,16 @@ class DevelopmentConfig(Config):
     }
     cachito_request_file_logs_dir: Optional[str] = "/var/log/cachito/requests"
     cachito_sources_dir = os.path.join(ARCHIVES_VOLUME, "sources")
+    if not os.environ.get("CACHITO_OTLP_EXPORTER_ENDPOINT"):
+        if os.environ.get("CACHITO_JAEGER_EXPORTER_ENDPOINT"):
+            print("Jaeger endpoint defined")
+            cachito_jaeger_exporter_endpoint = os.environ.get("CACHITO_JAEGER_EXPORTER_ENDPOINT")
+            cachito_jaeger_exporter_port = cast(int, os.environ.get("CACHITO_JAEGER_EXPORTER_PORT"))
+    else:
+        print(
+            "OTLP endpoint defined: " + cast(str, os.environ.get("CACHITO_OTLP_EXPORTER_ENDPOINT"))
+        )
+        cachito_otlp_exporter_endpoint = os.environ.get("CACHITO_OTLP_EXPORTER_ENDPOINT")
```

Comment:
Setting the attributes conditionally will cause the issue ilustrated in the comment above

I think all three of these need to be set in the base Config class and need to be set always

```python
cachito_jaeger_exporter_endpoint = os.environ.get("CACHITO_JAEGER_EXPORTER_ENDPOINT")
cachito_jaeger_exporter_port = os.environ.get("CACHITO_JAEGER_EXPORTER_PORT")
cachito_otlp_exporter_endpoint = os.environ.get("CACHITO_OTLP_EXPORTER_ENDPOINT")
```

I'm also a little nervous about reading environment variables at _import time_ (the code gets executed as soon as some part of Cachito imports cachito.workers.config). Is there a reason why these should be environment variables rather than being set directly in the code or in the config file (`prod_config_file_path`)? That's how configuration is usually handled.